### DISCUSSION TITLE
Create a deb package from Poetry project

### DISCUSSION THREAD

**Discussion by jrozhon** [python-poetry/poetry#9055] [Q&A]
Hi, I am struggling to find any examples of how to create a debian package from a project created with (most examples I found are setuptools-based). Is there please any example in the Poetry's community I can use?

I found that pybuild is probably what I want, but have not found any examples of how to use it.

https://manpages.debian.org/unstable/dh-python/pybuild.1.en.html
https://stackoverflow.com/questions/63304163/how-to-create-a-deb-package-for-a-python-project-without-setup-py
https://stackoverflow.com/questions/77557529/building-debian-python-package-with-poetry

Thank you

**Comment by abn** [comment]
Creating distro specific packages is not within the scope of Poetry unfortunately. But I am sure folks in the debian python community might have better ideas.

I did notice that there is a project called [wheel2deb](https://github.com/upciti/wheel2deb). You could simply do a `poetry build` and then pass the wheel to it to build your deb package perhaps?

**Reply by jrozhon** [reply]
Thank you for your answer. I am aware this is more like a debian-focused topic, but I had a feeling that inside of this community I might have a better chance of getting a relevant answer as in debian there is a mix of guys working with all the tools possible and still a majority of those relying on setuptools.
I will explore your link a bit more. Thank you.

**Comment by SovoN-Niko** [comment]
@jrozhon Hi, I have the same problem. My old working project was based on setuptools and used requirements.txt. Can you share how you solve your problem, please?

**Reply by jrozhon** [reply]
Hi, to be honest I went the full circle back to pure python distribution and docker. But in the interim, I found out a nice tool that handled the conversion pretty well. Only drawback is it is written in Ruby. Hope this helps. J

https://github.com/jordansissel/fpm

**Reply by SovoN-Niko** [reply]
Thanks))
